saleor/settings.py

```diff
@@ -598,6 +598,9 @@ def get_host():
 # Slug for channel precreated in Django migrations
 DEFAULT_CHANNEL_SLUG = os.environ.get("DEFAULT_CHANNEL_SLUG", "default-channel")
 
+# Set this to `True` if you want to skip creating default objects in database.
```

Comment:
Would be good to have more context here. Like what kind of the objects we will create etc. `default objects` is a little bit too generic.

Reply:
Yep, what default objects?